src/main/java/uk/gov/pay/adminusers/utils/date/DisputeEvidenceDueByDateUtil.java

```diff
@@ -1,24 +1,17 @@
 package uk.gov.pay.adminusers.utils.date;
 
-import java.time.Instant;
-import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 
 public class DisputeEvidenceDueByDateUtil {
-    
-    public static ZonedDateTime getZDTForEpoch(Long epoch) {
-        return Instant.ofEpochSecond(epoch).atZone(ZoneOffset.UTC);
-    }
-    
-    public static ZonedDateTime getPayDueByDateForEpoch(Long epoch) {
-        var zdt = getZDTForEpoch(epoch);
-        switch (zdt.getDayOfWeek()) {
+
+    public static ZonedDateTime getPayDueByDateForEpoch(ZonedDateTime evidenceDueDate) {
```

Comment:
Should this method be renamed now it isn't doing anything with epochs?

Reply:
Yeah, good spot! updated.